setup.py

```diff
@@ -81,23 +81,17 @@
     platforms='any',
     classifiers=[
         "Programming Language :: Python",
+        'Programming Language :: Python :: 2',
         "Programming Language :: Python :: 2.7",
-        # 'Development Status :: 1 - Planning',
-        # 'Development Status :: 2 - Pre-Alpha',
-        'Development Status :: 3 - Alpha',
-        # 'Development Status :: 4 - Beta',
-        # 'Development Status :: 5 - Production/Stable',
-        # 'Development Status :: 6 - Mature',
-        # 'Development Status :: 7 - Inactive',
-        "Environment :: Other Environment",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: Cython',
```

Comment:
I don't think ``Cython`` is appropriate here - it's not an implementation of Python, as it relies on either CPython or PyPy, depending on where it's used. But if you'll manage to make PyPy work, then you can add PyPy here as well.